Cannot start vagrant on Windows

### Description of problem

I've tried following [dokku/docs/getting-started/install/vagrant.md at master · dokku/dokku](https://github.com/dokku/dokku/blob/master/docs/getting-started/install/vagrant.md)

So I installed virtualbox 7.0 (7.1 is not accepted by Vagrant), and vagrant 2.4.1.

Running `env BOX_MEMORY=2048 vagrant up dokku-windows --provider=virtualbox` seems to work (when using master, switching to any branch fails), up to the following error:

```
    dokku-windows: apt-get update -qq
    dokku-windows: make: jq: Command not found
    dokku-windows: wget -qO /usr/local/bin/docker-image-labeler
    dokku-windows: wget: missing URL
    dokku-windows: Usage: wget [OPTION]... [URL]...
    dokku-windows:
    dokku-windows: Try `wget --help' for more options.
    dokku-windows: make: *** [Makefile:156: docker-image-labeler] Error 1
```

It seems jq is used in Makefile before it's available.

I believe it should be first in dependencies rule, or it should be a dependency of docker-image-labeler

### Steps to reproduce

Follow [[dokku/docs/getting-started/install/vagrant.md at master · dokku/dokku](https://github.com/dokku/dokku/blob/master/docs/getting-started/install/vagrant.md)](https://github.com/dokku/dokku/blob/master/docs/getting-started/install/vagrant.md) on a Windows host

### dokku report $APP_NAME

not available

### Additional information

_No response_

### Output of failing commands after running: dokku trace:off

_No response_

### Output of failing commands after running: dokku trace:on

_No response_

--- a/Makefile
+++ b/Makefile
@@ -131,7 +131,7 @@ plugin-dependencies: plugn procfile-util
 plugins: plugn procfile-util docker
 	sudo -E dokku plugin:install --core
 
-dependencies: apt-update docker-image-labeler docker-container-healthchecker lambda-builder netrc sshcommand plugn procfile-util docker help2man man-db sigil dos2unix jq parallel
+dependencies: apt-update jq docker-image-labeler docker-container-healthchecker lambda-builder netrc sshcommand plugn procfile-util docker help2man man-db sigil dos2unix parallel
 	$(MAKE) -e stack
 
 apt-update:

